### DISCUSSION TITLE
QuickHash - Brazilian Portuguese language translation

### DISCUSSION THREAD

**Discussion by GuzMan2019** [tedsmith/quickhash#105] [Ideas]
Hi Ted Smith

I can translate QuickHash GUI to Brazilian Portuguese language. I just need you to email me the strings.

Regards,

Guzmán

**Comment by tedsmith** [comment]
Sorry - I totally missed this! I'm just creating for v3.3.3 so if you are still interested let me know

**Comment by GuzMan2019** [comment]
Hi Smith,



Yes, you can send me all the texts in this version and I will translate them into Brazilian Portuguese.

Suggestion: make the application available for public download in .deb format



Regards,

Guzmán



________________________________
De: Ted Smith ***@***.***>
Enviado: sábado, 14 de outubro de 2023 15:08
Para: tedsmith/quickhash ***@***.***>
Cc: GuzMan2019 ***@***.***>; Author ***@***.***>
Assunto: Re: [tedsmith/quickhash] QuickHash - Brazilian Portuguese language translation (Discussion #105)


Sorry - I totally missed this! I'm just creating for v3.3.3 so if you are still interested let me know

**Reply by tedsmith** [reply]
Many thanks! It is a big manual though! You may not thank me. Anyway, the file is in fodt format titled 'UserManual.fodt' in the main branch : https://github.com/tedsmith/quickhash/tree/master
